catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/metatype/org.codice.ddf.catalog.ui.querymonitor.xml

```diff
@@ -23,6 +23,11 @@
             name="Query Timeout" id="queryTimeoutMinutes" required="true" type="Long"
             default="5"/>
 
+        <AD description="Set the interval in minutes for the query results in the workspace (Range: 0 - 1440 (24 hours)).
```

Comment:
Recommend renaming this to be something like `Relative Time Search (past X minutes)` to give an indication that this is searching the last X minutes, and this is the result set that will be included in the email.